lisp/mastodon-toot.el

```diff
@@ -222,7 +346,10 @@ If REPLY-TO-ID is provided, set the MASTODON-TOOT--REPLY-TO-ID var."
   "Minor mode to capture Mastodon toots."
   :group 'mastodon-toot
   :keymap mastodon-toot-mode-map
-  :global nil)
+  :global nil
```

Comment:
That is the default anyway, right?  Is there added value in explicitly stating this?